keyboards/wheatfield/split75/split75.h

```diff
@@ -28,6 +28,24 @@
   { K07,    K17,    K27,    K37,    K47,    KC_NO,  KC_NO,  KC_NO,  K87,    K97,    K107,   K117,   K127,   K137 }  \
 }
 
+#define LAYOUT_ISO( \
+            K05, K15, K25, K35, K45, K55, K65, K75, K85, K95, K105, K115, K125, K135, K86, K87, \
+  K47, K46, K04, K14, K24, K34, K44, K54, K64, K74, K84, K94, K104, K114, K124,       K96, K97, \
+  K37, K36, K03, K13, K23, K33, K43, K53,      K73, K83, K93, K103, K113, K123, K133, K116, K107, \
+  K27, K26, K02, K12, K22, K32, K42, K52,      K72, K82, K92, K102, K112, K122, K132,       K117, \
```

Comment:
QMK custom is that the Enter keycode for ISO/JIS layouts should be toward the end of the home row.

```suggestion
  K37, K36, K03, K13, K23, K33, K43, K53,      K73, K83, K93, K103, K113, K123, K133,       K107, \
  K27, K26, K02, K12, K22, K32, K42, K52,      K72, K82, K92, K102, K112, K122, K132, K116, K117, \
```